components/brave_wallet/common/brave_wallet_types.h

```diff
@@ -157,6 +157,8 @@ struct SolanaAccountInfo {
   uint64_t rent_epoch;
 };
 
+extern const char kMetamaskExtensionId[];
```

Comment:
can we keep `inline constexpr char kMetamaskExtensionId[]` which is more modern and not occupied memory when fully optimized at compile time?